test/e2e/uninstall_kubelet_test.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/openshift/windows-machine-config-bootstrapper/pkg/bootstrapper"
+)
+
+func TestKubeletUninstall(t *testing.T) {
+	t.Run("Uninstall Kubelet", testKubeletUninstall)
+}
```

Comment:
I think you can take this one step further and remove this function and rename `testUninstallWithoutKubeletSvc` to `TestUninstallWithoutKubeletSvc`

Reply:
I think you meant TestKubeletUninstall, I have already modified TestBootstrapper/testUninstallWithoutKubeletSvc